pyproject.toml

```diff
@@ -17,6 +19,8 @@ classifiers = [
     "Programming Language :: Python :: 3.8",
     "Programming Language :: Python :: 3.9",
     "Programming Language :: Python :: 3.10",
+    "Programming Language :: Rust",
+    "Programming Language :: Python :: Implementation :: CPython",
```

Comment:
```suggestion
```

Reply:
github gui "commit suggestion" was buggy so made a manual commit for this change.